scipp/whats-new-0.5.0.ipynb

```diff
@@ -0,0 +1,342 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What's new in scipp 0.5.0?\n",
+    "\n",
+    "[Release Notes](https://scipp.github.io/about/release-notes.html#v0-5-january-2021)\n",
+    "\n",
+    "Due to two big internal and external changes this release cycle has been very long, with many significant changes.\n",
+    "We generally aim for shorter cycles in the future.\n",
+    "\n",
+    "Download [PG3_4844_event.nxs](http://198.74.56.37/ftp/external-data/MD5/d5ae38871d0a09a28ae01f85d969de1e) (and rename after download) to run this notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run logs now stored natively\n",
+    "\n",
+    "- Previously we stored Mantid's `Run` and `Sample` as a scalar variable holding the Mantid object as a `PyObject`.\n",
+    "- `Run` is now converted into individual attributes.\n",
+    "- `Sample` still `PyObject`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = 'PG3_4844_event'\n",
+    "data = sc.neutron.load(filename=f'{filename}.nxs')\n",
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Many time-dependent logs are just data arrays, so we can access, modify, plot, ... them.\n",
+    "Remember that the `value` property has to be used to access the only value of a scalar variable:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.plot.plot(data.attrs['proton_charge'].value, marker='.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Improved instrument view\n",
+    "\n",
+    "- Align camera to axis buttons\n",
+    "- Cut surface tools\n",
+    "- Bug fixes, such as pixel sizes on high-res displays"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.neutron.instrument_view(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Improved plotting\n",
+    "\n",
+    "- `logx`, `logy`, `log`, and transpose buttons in toolbar\n",
+    "- now available as method in addition to free function, `data.plot()` instead of `sc.plot.plot(data)`\n",
+    "- profile plot/picking for 3-D (and higher) data\n",
+    "- Cleaned up widgets for 3-D (and higher) data\n",
+    "- Performance improvements\n",
+    "- Dynamic event data resampling (see below)"
```

Comment:
I think the `aspect` functionality is new too.

Reply:
Seems to be missing from the release notes.